src/Qt-RSA/qrsaencryption.cpp

```diff
@@ -290,20 +289,76 @@ QByteArray decodeArray(const QByteArray &rawData, const QByteArray &privKey) {
     return res.remove(res.lastIndexOf(ENDLINE), res.size());
 }
 
-bool QRSAEncryption::testKeyPair(const QByteArray &pubKey, const QByteArray &privKey) {
-    QByteArray tesVal = "Test message of encrypkey";
+QRSAEncryption::QRSAEncryption(Rsa _keyLength)
+{
+    keyLength = _keyLength;
+}
 
-    bool result = tesVal == decode(encode(tesVal, pubKey), privKey);
+unsigned int QRSAEncryption::getBytesSize(QRSAEncryption::Rsa keyLength) {
+    return keyLength / 8;
+}
 
-    if (!result) qWarning() << "(Warning): Testkey Fail, try generate new key pair!";
+// --- static methods ---
+bool QRSAEncryption::generatePairKey(QByteArray &pubKey, QByteArray &privKey,
+                                     QRSAEncryption::Rsa keyLength) {
 
-    return result;
+    return QRSAEncryption(keyLength).generatePairKey(pubKey, privKey);
```

Comment:
everything is correct here, but since the constructor with the bitness is no longer present, we pass the bitness to the 3rd parameter